game/scripts/npc/items/item_octarine_core.txt

```diff
@@ -47,7 +47,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_intelligence"                              "25 38 57 85 115"
+        "bonus_intelligence"                              "25 40 65 100 145"
```

Comment:
Revert this. Octarine Core already gives more INT than it should.

Reply:
Normalised the scaling. Goes +15 +20 +25 +30 INT. Same levels 4 and 5 but better levels 2 and 3 by 2 and 3 INT respectively.